doc/source/enhancements.rst

```diff
@@ -1,3 +1,5 @@
+.. _chapter_enhancements:
+
```

Comment:
This shouldn't be needed. I think you can do `:doc:'enhancements'` and it will serve the same purpose.

Reply:
Good to know. Will change it.